getting-started/windows-setup.md

```diff
@@ -116,29 +47,6 @@ ln -s /mnt/c/Users/{%USER NAME} ms-home
 ls -l
 ```
 
-## Improving the WSL terminal experience
```

Comment:
I decided to cut this; it's beyond the scope of a basic getting started. I understand WSL terminal might feel a bit clunky but the cost of including, explaining, and supporting this section seems like too much outside of the bootcamp.

Reply:
Sounds good